tests/postgres_tests/test_integration.py

```diff
@@ -0,0 +1,23 @@
+import os
+import subprocess
+import sys
+import unittest
+
+from django.test import SimpleTestCase
+
+try:
+    import psycopg2
+except ImportError:
+    psycopg2 = None
+
+
+@unittest.skipIf(psycopg2 is None, 'psycopg2 is required')
+class PostgresIntegrationTest(SimpleTestCase):
+    def test_management_command(self):
+        manage_py = os.path.join(os.path.dirname(__file__), 'integration', 'manage.py')
+        result = subprocess.run(
+            [sys.executable, manage_py, 'check'],
```

Comment:
You can call `['python', '-m', 'django', '--settings', 'integration_settings', 'check']` instead to avoid creating a dedicated `manage.py` file.

Reply:
In the suggested command list, `'python'` should `sys.executable` to ensure the same Python that is running the tests is used in the subprocess.